runtime-modules/staking-handler/Cargo.toml

```
@@ -1,23 +1,25 @@
 [package]
 name = 'pallet-staking-handler'
-version = '2.0.0'
+version = '5.0.0'
```

Version incorrectly updated
```suggestion
version = '3.0.0'
```